plugins/network-elements/nsx/src/main/java/org/apache/cloudstack/service/NsxApiClient.java

```diff
@@ -111,6 +116,10 @@ public class NsxApiClient {
     protected static final String SEGMENTS_PATH = "/infra/segments";
     protected static final String DEFAULT_DOMAIN = "default";
     protected static final String GROUPS_PATH_PREFIX = "/infra/domains/default/groups";
+    // TODO: Pass as global / zone-level setting?
+    protected static final String NSX_LB_PASSIVE_MONITOR = "/infra/lb-monitor-profiles/default-passive-lb-monitor";
```

Comment:
In that case, only this one should be a setting or other paths above as well?

Reply:
Honestly for everything, we'd need a better way of handling it that hard-coding it.